### ISSUE TITLE
Directly fill `event.lst.tel[X].first_capacitor_id` with the correct array

### ISSUE THREAD

**Issue by maxnoe** [cta-observatory/ctapipe_io_lst#118]
Instead of storing the `first_caps` array in the `LSTR0Corrections`, we can just put the correct array of shape `(N_GAIN, N_PIXELS)` into the event.

This will make it accessible also to the scripts in lstchain that at the moment recalculate it.

**Comment by FrancaCassol** [comment]
Hi @maxnoe,
this implies a request  to change the data model, for the moment only the "first cell id" is foreseen

**Comment by maxnoe** [comment]
No, it does not change anything about the R1 data model. It would be completely internal to the analysis software.
